packages/functions/lambdas/slack-interaction.ts

```diff
@@ -44,14 +53,47 @@ export const handler: APIGatewayProxyHandlerV2 = async (request) => {
         if (!presentIdea) {
           throw new Error("Present idea is empty");
         }
+        await publishEvent("savePresentIdea", {
+          birthdayPerson: parsedData.actions[0].value,
+          user: parsedData.user.id,
+          team: parsedData.user.team_id,
+          presentIdea,
+          responseUrl: parsedData.response_url,
+        });
+        break;
+      }
+      case additionalPresentIdeasSaveButtonActionId: {
+        const presentIdea =
+          parsedData.state?.values?.[additionalPresentIdeasInputBlockId]?.[
+            additionalPresentIdeasInputActionId
+          ]?.value;
+        const squadJoin =
+          !!parsedData.state?.values?.[squadJoinCheckboxBlockId]?.[
+            squadJoinCheckboxActionId
+          ]?.selected_options?.length;
 
+        console.log({ presentIdea, squadJoin });
+        if (!presentIdea) {
+          throw new Error("Additional Present idea is empty");
+        }
+        if (squadJoin) {
+          await publishEvent("saveSquadJoin", {
+            birthdayPerson: parsedData.actions[0].value,
+            user: parsedData.user.id,
+            team: parsedData.user.team_id,
+          });
+        }
         await publishEvent("savePresentIdea", {
           birthdayPerson: parsedData.actions[0].value,
           user: parsedData.user.id,
           team: parsedData.user.team_id,
           presentIdea,
           responseUrl: parsedData.response_url,
         });
+        break;
+      }
+      case squadJoinCheckboxActionId: {
+        console.warn("Warning Squad join checkbox action");
```

Comment:
Could you please add a comment that this doesn't need to be handled? It looks like it's a todo or sth.